src/main/java/org/fundacionjala/core/util/Assert.java

```diff
@@ -11,9 +11,6 @@
 
     private Assertion assertion;
 
-    /**
-     * Constructor.
-     */
```

Comment:
Why are you deleting this javadoc ?

Reply:
It was a problem with merge. its already fixed